ocaml/ocaml-pcre2/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           ocaml 1.1
+PortGroup           github 1.0
+
+name                ocaml-pcre2
+version             7.5.2
+revision            0
+categories          ocaml devel
+maintainers         {pguyot @pguyot} openmaintainer
+license             LGPL
```

Comment:
this should be `LGPL-2.1`

Reply:
Thanks. I couldn't find a list of licenses, maybe we should stick to SPDX?

Reply:
no, we should use a license that MacPorts recognized as it is used to decide whether a port is distributable (see: https://github.com/macports/macports-infrastructure/blob/d467a2b8e417742baf678042f9f93c392a45dcac/jobs/distributable_lib.tcl).

Reply:
Thanks. I searched for the list in base. I'll add it to port lint eventually.